backend/main/src/User/Controllers/CreateUserController.ts

```diff
@@ -5,6 +5,7 @@ import { CreateUserUseCase } from "src/User/UseCases/CreateUserUseCase/CreateUse
 interface ICreateUserInput {
 	id: string;
 	name: string;
```

Comment:
I'm sorry it's not related to this PR topic directly, but I think this name prop can be optional

Reply:
It should be not needed. So, I deleted it in the "update user ID" PR.

Reply:
Lol. my bad 🤣